### ISSUE TITLE
Only having salt makes it easy to add incorrect sodium/salt quantities to OFF

### ISSUE THREAD

**Issue by user234683** [davidhealey/waistline#527]
When scanning US items, I was not aware of the distinction between salt (the EU measurement) and sodium (the US measurement). So when I scanned a US item that wasn't in OFF, I figured I should put the sodium value into the salt box, resulting in incorrect data for a few items. I think the sodium value should be placed next to the salt box by default rather than being hidden under "More nutriments" in order to prevent bad data being put into OFF by US users.

**Comment by EmilJunker** [comment]
Users can customize which nutrients are be displayed and also change their order in settings. I would expect users from the USA to make use of this and to replace the salt field with the one for sodium.

Showing both salt and sodium by default sounds like a bad idea to me. But I agree that we should at least change the default order and place them right next to each other.

**Comment by davidhealey** [comment]
What about using the users locale to decide the default?

**Comment by EmilJunker** [comment]
Yes, that could work. Alternatively, maybe we could allow the user to choose between salt and sodium as part of the planned setup wizard (along with some other settings like preferred units and so on).

**Comment by user234683** [comment]
> I would expect users from the USA to make use of this and to replace the salt field with the one for sodium.

This is only if they are aware of the distinction, but my point is most people are not aware of it, so they will provide OFF with incorrect data (which is what I did)

> What about using the users locale to decide the default?

I think this is a great idea
